app/lib/actions/katello/repository/refresh_repostiory.rb

```diff
@@ -0,0 +1,13 @@
+module Actions
+  module Katello
+    module Repository
+      class RefreshRepository < Actions::Base
+        def plan(repo)
+          ::User.current = ::User.anonymous_admin
```

Comment:
The way we normally ensure the current user is set is either:

a) use a `User.as_anonymous_admin` blocktask
b) use the KeepCurrentUser middleware, and set the user outside of the (probably using a)

I tend to prefer a) as it keeps the user setting out of the tasks.